src/utils/environment.rs

```diff
@@ -8,7 +8,7 @@ pub fn fetch_var(name: &str, default: &str) -> String {
       if value.is_empty() {
         String::from(default)
       } else {
-        value
+        String::from(value.trim_start_matches('"').trim_end_matches('"'))
```

Comment:
I don't know if I like this behavior. This will lead to unexpected behavior if we ever have an environment variable that's supposed to start and/or end with double-quotes